test/test.js

```diff
@@ -19,4 +19,16 @@ describe('is-relative-path', function () {
     assert.ok(!isRelative('foo'));
     assert.ok(!isRelative('foo/bar/car/baz.js'));
   });
+
+  it('throws on empty input', function () {
+    assert.throws(function () {
+      isRelative();
+    }, function (err) { return (err instanceof TypeError); });
```

Comment:
Would he helpful to also test that error.message is what we expect. That rules out typos or spacing issues in the generated error message.